rust/ql/test/query-tests/unusedentities/main.rs

```diff
@@ -322,7 +391,58 @@ fn shadowing() -> i32 {
     }
 }
 
+// --- function pointers ---
+
+type FuncPtr = fn(i32) -> i32;
+
+fn increment(x: i32) -> i32 {
+    return x + 1;
+}
+
+fn func_ptrs() {
+    let MyFunc: FuncPtr = increment;
```

Comment:
```suggestion
    let my_func: FuncPtr = increment;
```
Just to make it `snake_case` like Rust functions usually are.

Reply:
Good point.  I've accepted your suggestions.